langtrace/langtrace-elastic-demo.py

```diff
@@ -0,0 +1,84 @@
+import os
+import asyncio
+from dotenv import load_dotenv
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from langtrace_python_sdk import langtrace, with_langtrace_root_span
+from langchain_openai import AzureChatOpenAI
+from langchain.prompts import ChatPromptTemplate
+from langchain.schema.runnable import RunnableSequence
+from langchain_community.tools import DuckDuckGoSearchResults
+from langchain_community.utilities import DuckDuckGoSearchAPIWrapper
+
+# Load environment variables
+load_dotenv()
+
+# Elastic APM settings
+ELASTIC_APM_SERVER_URL = os.getenv("ELASTIC_APM_SERVER_URL")
+SERVICE_NAME = os.getenv("SERVICE_NAME", "langtrace-elastic-demo")
+ELASTIC_APM_SECRET_TOKEN = os.getenv("ELASTIC_APM_SECRET_TOKEN")
+
+# Set up Elastic exporter
+elastic_exporter = OTLPSpanExporter(
+    endpoint=ELASTIC_APM_SERVER_URL,
+    headers={"Authorization": f"Bearer {ELASTIC_APM_SECRET_TOKEN}"}
+)
+
+# Initialize Langtrace with Elastic exporter
+langtrace.init(
+    custom_remote_exporter=elastic_exporter,
+    batch=True,
+    service_name=SERVICE_NAME
+)
+
```

Comment:
Will this langtrace.init work if we set up the OTel env variables outside the code in the shell and run 

opentelemetry-instrument python nameoffile.py

Reply:
Yes langtrace init will work if you setup OTel ENV variables outside of the code. I will modify the code to show this

Reply:
Hey @bshetti  - I've updated the PR with ENV somewhat outside of code. We are working on making this even better so you don't have to specify the ENV in code at all. Let me know if you run into any issues.